src/main/scala/com/cognite/sdk/scala/v1/resources/sequenceData.scala

```diff
@@ -0,0 +1,3 @@
+package com.cognite.sdk.scala.v1.resources
+
+class sequenceData {}
```

Comment:
Looks like this is not in use.

Reply:
Yep, good catch. I'll work on it next so it might as well go in with this commit.